content/blog/2022-01-15-moving-local-experiments-to-the-cloud-with-iterative-terraform-provider.md

```diff
@@ -38,11 +38,11 @@ Terraform and run your existing training script on it.
 
 For this tutorial I have picked
 [The BeeImage Dataset](https://www.kaggle.com/jenny18/honey-bee-annotated-images)
-that contains 5,100+ bee images annotated with location, date, time, subspecies,
+which contains over 5,100 bee images annotated with location, date, time, subspecies,
 health condition, caste, and pollen. I've downloaded the images, created a
 project and trained a simple CNN model locally to classify different subspecies.
 If you want to follow along, you can use your own data and training code, or
-clone [the example repo from GitHub](https://github.com/MKhalusova/bees).
+clone [the example repo from GitHub](https://github.com/iterative/bees).
```

Comment:
I wonder - should we have a repo naming & topic convention? Maybe:

repo name | repo [topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics) | description
--|--|--
`example-*` | `demo`, `docs` | used in official docs
`blog-*` | `demo`, `blog` | used in blog posts
`course-*` | `demo`, `course` | used in official courses
`test-*` | `demo`, `test` | debugging & CSE

Alternatives names & topics which are also inconsistently used include `demo`, `learn`, `eg`, `intro`, `scenario`, `lesson`.

Repo [topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics) make it easy to find/filter repos and are probably more important than repo name. [iterative topic:example](https://github.com/iterative?q=topic%3Aexample&type=source) lists only one repo atm :cry:

There are ~150 repos in total and it's quickly getting impossible to keep track.

/CC @jorgeorpinel @iesahin @shcheklein @flippedcoder @mnrozhkov

Reply:
I began to believe having a dedicated Github organization like `iterative-examples` might be a better approach for the example repos. We can fork an example repository to a new name, modify it and use in a blog post/document. At least the fork lineage becomes trackable that way, and we can keep the tags, READMEs and whatnot. 

Personally, I like the current approach better, but planning them centrally and building them with a script (like `example-repos-dev`) seems not scalable when teams have deadlines to meet.

Reply:
1. I believe this is a discussion that needs to be had elsewhere so that the decision isn't lost in the comments to a particular PR.
2. I moved the repo under Iterative as requested, however, if we want to create a plethora of content with examples, having all example repos under Iterative is not sustainable. In my opinion, the examples for the official course and documentation (if needed) should be under Iterative. Everything else - examples for blog, talks, workshops, meetups, and so on are fine to be under personal accounts. These are meant to be temporary.

Reply:
@shcheklein After I transferred the repo to Iteartive, I no longer have access to its settings. Please rename however you feel appropriate and I'll update the link in the blog post.

Reply:
done! https://github.com/iterative/blog-tpi-bees

Reply:
Thanks! Updated the link in b818609d7accb5befc43f0b20477793c426955c5

Reply:
Follow-up ticket at #3151